hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/discovery/ConfigurationXmlEntry.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.discovery;
+
+import javax.xml.bind.annotation.XmlElement;
+
+/**
+ * JAXB representation of one property of a hadoop configuration XML.
+ */
+public class ConfigurationXmlEntry {
+
+  @XmlElement
+  private String name;
+
+  @XmlElement
+  private String value;
```

Comment:
Don't we also have Tags and Comments ?

Reply:
We have but:

1.) I am not sure if they should be added to the downloaded file. (Usually we don't add tag/description to an ozone-site.xml). But it can be useful to add tags/descriptions to the generated file just to make it easier to identify the used configuration.

2.) Unfortunately the Configuration class is very limited. AFAIK I can't get all the available tags (the key set of propertyTagsMap) and I can't see if the descriptions are loaded at all... 

I may be wrong but I think first we should improve the Configuration.java on trunk, and get a 3.3 hadoop release OR fork more methods from Configuration and copy them to the OzoneConfiguration.

Reply:
ok, Forget that I made this comment.  :(